app/views/stories/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<p>
+  <strong>Name:</strong>
+  <%= @story.name %>
+</p>
+
+<p>
+  <strong>Address:</strong>
+  <%= @story.address %>
+</p>
+
+<p>
+  <strong>Category:</strong>
+  <%= @story.category %>
+</p>
+
+<p>
+  <strong>Rating:</strong>
+  <%= @story.rating %>
+</p>
+
+<p>
+  <strong>Stuff type:</strong>
+  <%= @story.stuff_type %>
+</p>
+
+<p>
+  <strong>Story:</strong>
+  <%= @story.description %>
+</p>
+
+<div>
+  <strong>Attachments:</strong>
+  <br>
+  <% @story.images.each do |img| %>
```

Comment:
why do we need so many conditions here? can't we restrict the uploading to be just images?

Reply:
The requirements allow images/audio/video.
Created a partial _display_uploads to house the shared code
renamed images to uploads